azure/table_azure_lb.go

```diff
@@ -0,0 +1,276 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/profiles/2020-09-01/monitor/mgmt/insights"
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2021-02-01/network"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+)
+
+//// TABLE DEFINITION ////
+
+func tableAzureLoadBalancer(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_lb",
+		Description: "Azure Load Balancer",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.AllColumns([]string{"name", "resource_group"}),
+			Hydrate:           getLoadBalancer,
+			ShouldIgnoreError: isNotFoundError([]string{"ResourceNotFound", "ResourceGroupNotFound", "404"}),
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listLoadBalancers,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "Resource name.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "id",
+				Description: "Resource ID.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromGo(),
+			},
+			{
+				Name:        "provisioning_state",
+				Description: "The provisioning state of the load balancer resource. Possible values include: 'ProvisioningStateSucceeded', 'ProvisioningStateUpdating', 'ProvisioningStateDeleting', 'ProvisioningStateFailed'",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("LoadBalancerPropertiesFormat.ProvisioningState").Transform(transform.ToString),
```

Comment:
Do we need Transform(transform.ToString) ?

Reply:
It will come as `null` otherwise

Reply:
I understood the comment wrong. I have made the changes